port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
-    ServerAdmin          admin@mycoolsite.com
```

Comment:
Setting the ServerAdmin name. It's a valid parameter but honestly, I have no idea where it gets used! I'd probably rather have it than not though?